print-esp.c

```diff
@@ -332,6 +332,20 @@ int espprint_decode_hex(netdissect_options *ndo,
 	return i;
 }
 
+static int
+strendswith(char *str, char *end)
+{
+	char *p;
+
+	if (strlen(str) > strlen(end) &&
+	    !strcmp(str + strlen(str) - strlen(end), end)) {
```

Comment:
perhaps a comment or two here explaining the logic.   A temp variable or two for strlen(XX) would also increase readability.

Reply:
Sure, thanks!